common/workunit/workunit.cpp

```diff
@@ -2433,8 +2470,21 @@ WuScopeFilter & WuScopeFilter::addFilter(const char * filter)
                 else
                     scopeFilter.setDepth(low, UINT_MAX);
             }
+            else if (dotdot)
+            {
+                unsigned low = atoi(arg); // note: =0 if number omitted
+                if (dotdot[2])
```

Comment:
@ghalliday Invalid values not checked for.  Given that invalid values causes an exception to be thrown in line 2486, same behavour should be here.  Otherwise,  invalid values containing comma or dotdot does not cause an exception but a single value does - this is inconsistant.

Reply:
What examples are you thinking of

[z..]?